DESCRIPTION

```diff
@@ -45,6 +47,8 @@ Suggests:
     covr,
     knitr,
     ncdf4,
+    pkgconfig,
+    quitte,
```

Comment:
I think adding `quitte` here creates problems when updating `magclass` on CRAN as `quitte` is not available there. Can you remove it again from the `suggests` list or is it required?

Reply:
I added it because pre-commit.ci was complaining…

Reply:
ah ok. I thought it was only complaining about pkgconfig. If both was needed, then we can keep it for the moment and I will find out with my next CRAN-update whether this is accepted there or not.

Reply:
See https://results.pre-commit.ci/run/github/93050064/1666187376.mH43gdB9QXCEJrhPgUS4Jw
```
Not all packages used in your code are listed in DESCRIPTION. The following are missing in file `/code/tests/testthat/test-as.magpie.R`: quitte.
```
On the other hand, I'm not sure what were just warnings and what lead to failure.

Reply:
strictly speaking, quitte is not a dependency, because its use is guarded by checks if it is installed. So, technically, we don't need any kind of dependency, but a suggests also doesn't feel wrong.

Reply:
"Packages on which a CRAN package depends should be available from a mainstream repository: if any mentioned in ‘Suggests’ or ‘Enhances’ fields are not from such a repository, where to obtain them at a repository should be specified in an ‘Additional_repositories’ field of the DESCRIPTION file (as a comma-separated list of repository URLs) or for other means of access, described in the ‘Description’ field." ([source](https://cran.r-project.org/web/packages/policies.html)).

Maybe this is an option? Or submit `quitte` to CRAN?

Reply:
ah cool....so Additional_repositories might then in fact be a feasible solution.

@mikapfl There is no doubt that it makes sense to mention it, but it was more the issue with the checks which come when you upload it to CRAN.